packages/amplify-codegen-e2e-tests/src/__tests__/add-codegen-js.test.ts

```diff
@@ -22,7 +22,7 @@ const schema = 'simple_model.graphql';
 
 describe('codegen add tests - JS', () => {
     let projectRoot: string;
-    const config = DEFAULT_JS_CONFIG;
+    let config = DEFAULT_JS_CONFIG;
```

Comment:
Why make this change?

Reply:
Oops this is made by mistake. I will fix this in follow up PR